src/SFA.DAS.EmployerIncentives.Data/ApprenticeApplicationDataRepository.cs

```diff
@@ -66,7 +66,8 @@ public async Task<List<ApprenticeApplicationDto>> GetList(long accountId, long a
                         InLearning = InLearning(data.learner),
                         PausePayments = data.incentive.PausePayments,
                         PaymentSent = data.firstPaymentSent != null,
-                        PaymentSentIsEstimated = IsPaymentEstimated(data.firstPaymentSent, _dateTimeService)
+                        PaymentSentIsEstimated = IsPaymentEstimated(data.firstPaymentSent, _dateTimeService),
+                        RequiresNewEmployerAgreement = !data.account.SignedAgreementVersion.HasValue || data.account.SignedAgreementVersion < data.incentive.MinimumAgreementVersion
```

Comment:
should this logic be in the domain layer?

Reply:
good point.  That would need all the ApprenticeshipIncentives for a legal entity to be updated when the legal entity SignedAgreementVersion is changed and also storing a RequiresNewEmployerAgreement flag against the ApprenticeshipIncentive and updatting existing ones.  So quite a big change.  This way it's a read view of the model.  

Could make the above change as a tech debt ticket perhaps.  Speak to Mike?

Reply:
After speaking to Mike have created tech debt ticket 

https://skillsfundingagency.atlassian.net/browse/EI-1147